docs/api/ref/DesktopAgent.md

```diff
@@ -645,14 +655,14 @@ raiseIntentForContext(context: Context, app?: AppIdentifier): Promise<IntentReso
 
 Finds and raises an intent against apps registered with the desktop agent based purely on the type of the context data.
 
-The desktop agent SHOULD first resolve to a specific intent based on the provided context if more than one intent is available for the specified context. This MAY be achieved by displaying a resolver UI. It SHOULD then resolve to a specific app to handle the selected intent and specified context. 
-Alternatively, the specific app or app instance to target can also be provided, in which case any method of resolution SHOULD only consider intents supported by the specified application. 
+The Desktop Agent SHOULD first resolve to a specific intent based on the provided context if more than one intent is available for the specified context. This MAY be achieved by displaying a resolver UI. It SHOULD then resolve to a specific app to handle the selected intent and specified context.
```

Comment:
```suggestion
The desktop agent SHOULD first resolve to a specific intent based on the provided context if more than one intent is available for the specified context. This MAY be achieved by displaying a resolver UI. It SHOULD then resolve to a specific app to handle the selected intent and specified context.
```